torch/testing/_internal/common_methods_invocations.py

```diff
@@ -80,6 +81,20 @@ def _getattr_qual(obj, name, default=_NOTHING):
 class OpInfo(object):
     """Operator information and helper functions for acquiring it."""
 
+    class AliasInfo(object):
+        """Placeholder for aliases. For example, torch.abs -> 
```

Comment:
Instead of "Placeholder for aliases", which is ambiguous, I think this can just say that it acquires and holds alias information.